cli/root.go

```diff
@@ -83,10 +110,78 @@ func init() {
 		// override flags if exist
 		cfg.Flags = override
 
-		// register config
-		err := Container.Register(cfg)
+		endureCfg := initEndureConfig()
+		if endureCfg == nil {
+			panic("endure config should not be nil")
```

Comment:
This is BC, container config should be optional for the user. All the current configs in all the apps and demos will have to be updated.

Reply:
Oh, I see.

Reply:
Yeah, this is not a BC. Endure section is optional. Check for nil here is just for the sake of a future updates